docs/content/configuration/global-configuration/command-line-arguments.md

```diff
@@ -536,6 +536,14 @@ The default value is `true`.
 
 <a name="cmdoption-ssl-dynamic-reload"></a>
 
+### -enable-telemetry-reporting
+
+Enable gathering and reporting of product related telemetry.
```

Comment:
```suggestion
Enable gathering and reporting of software telemetry.
```

Small thing, but the word product has marketing ick attached to it. No need to mention "related" since there's an assumption it is related to the software you're configuring.